cargo-shuttle/tests/integration/run.rs

```diff
@@ -5,10 +5,24 @@ use std::{fs::canonicalize, process::exit, time::Duration};
 use tokio::time::sleep;
 
 /// creates a `cargo-shuttle` run instance with some reasonable defaults set.
-async fn cargo_shuttle_run(working_directory: &str) -> u16 {
+async fn cargo_shuttle_run(working_directory: &str, router_ip: bool) -> u16 {
     let working_directory = canonicalize(working_directory).unwrap();
+
+    let url = if !router_ip { "localhost" } else { "0.0.0.0" };
```

Comment:
Maybe also return the url (or full address) from this function now

Reply:
Ok that makes sense, I'll update and send the changes through. Should not be too much of an issue

Reply:
Amended - please see new commit

Reply:
Nice :partying_face: